tests/pipelines/test_pipelines_automatic_speech_recognition.py

```diff
@@ -266,6 +262,49 @@ def test_torch_whisper(self):
         output = speech_recognizer([filename], chunk_length_s=5, batch_size=4)
         self.assertEqual(output, [{"text": " A man said to the universe, Sir, I exist."}])
 
+    @require_torch
+    @slow
+    def test_whisper_timestamp_prediction(self):
+        processor = WhisperProcessor.from_pretrained("openai/whisper-tiny")
+        model = WhisperForConditionalGeneration.from_pretrained("openai/whisper-tiny")
+        model.config.forced_decoder_ids = processor.get_decoder_prompt_ids(
+            task="transcribe", language="en", no_timestamps=False
+        )
+        model.config.begin_suppress_tokens += [50363]
+        ds = load_dataset("hf-internal-testing/librispeech_asr_dummy", "clean", split="validation").sort("id")
+        array = np.concatenate(
+            [ds[40]["audio"]["array"], ds[41]["audio"]["array"], ds[42]["audio"]["array"], ds[43]["audio"]["array"]]
+        )
+        pipe = pipeline(
+            "automatic-speech-recognition",
+            model=model,
+            tokenizer=processor.tokenizer,
+            feature_extractor=processor.feature_extractor,
+            forced_bos_token_id=None,
+            max_new_tokens=448,
+        )
```

Comment:
This seems way too complex to me.

We shouldn't need the Processor,  `pipeline(model=model)` should work out of the box.
`get_decoder_prompt_ids` seems like a method that should belong in the tokenizer (it does treat with ids and text, and nothing audio like it seems)

The 2 config modifications are kind of impossible to discover for a regular user no ? You pretty much HAVE to copy from somewhere. no ? 
We should at the very least comment what the intent is for those lines, or what would happen if we didn't put them

I'm gessing something like
```
# We need to start the decoding with whisper differently to enable the model to output timestamp like tokens
```
I have no idea what `begin_suppress_tokens` is supposed to be.